docker/syslog/Dockerfile

```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.9.7.24076
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
+  libffi-dev openssl-dev \
```

Comment:
```suggestion
  libffi-dev \
```

Reply:
openssl is actually needed, tagged u in the slack thread regarding it

Reply:
@wissamg fyi